tests/sources/test_github.py

```diff
@@ -600,6 +615,18 @@ async def test_validate_config_with_invalid_token_then_raise():
             await source.validate_config()
 
 
+@pytest.mark.asyncio
+async def test_validate_config_with_extra_scopes_token():
+    async with create_github_source() as source:
+        source.github_client.post = AsyncMock(
+            return_value=(
+                {"user": "username"},
+                {"X-OAuth-Scopes": "user, repo, admin:org"},
+            )
+        )
+        assert await source.validate_config() is None
```

Comment:
```suggestion
async def test_validate_config_with_extra_scopes_token(patch_logger):
    async with create_github_source() as source:
        source.github_client.post = AsyncMock(
            return_value=(
                {"user": "username"},
                {"X-OAuth-Scopes": "user, repo, admin:org"},
            )
        )
        assert await source.validate_config()
        patch_logger.assert_present("Configured token does not have required rights to fetch the content. Required scopes are 'repo', 'user', and 'read:org'.")
```

Reply:
Addressed in https://github.com/elastic/connectors/pull/1997/commits/6f3db163890fc46f2355bdcf0be878c67a679f83